packages/cli/src/commands/up.ts

```diff
@@ -51,15 +52,20 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
-    const { hostKey, clientId, rootUrl } = await tunnelServerHello({
-      tunnelingKey,
-      knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelOpts,
-      log: this.logger,
-    })
+    const { clientId, rootUrl, hostKey } = await withSpinner(async spinner => {
+      spinner.text = 'Initializing tunnel connection...'
```

Comment:
As [mentioned](https://github.com/livecycle/preevy/pull/148#discussion_r1282665645) this is misleading, there is no "connection", it's just a single API call.

```suggestion
      spinner.text = 'Getting tunnel server details'
```